### PR TITLE
Play updates

### PR DESCRIPTION
**Pull request by jordan-emma**
Play page reformatted, toggle join game and host game function removed and replaced with toggle game mode. Try catch blocks implemented on async functions. Main.js refactored to separate toast and socket concerns.

### CODE REVIEW

File: src/js/socket.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { io } from 'socket.io-client';
+
+export const socket = io.connect('http://localhost:5000');
```
Comment (HaydenNorris) [inline src/js/socket.js:3]: we will need to update this to use the env. The original code used:

import.meta.env.VITE_API_BASE_URL

File: src/main.js
Comment (HaydenNorris) [inline src/main.js:1]: so much cleaner!

File: src/views/PlayView.vue [hunk 2]
```diff
@@ -2,24 +2,38 @@
   <div class="purpleBackground">
     <div class="pageContainer">
       <div class="back">
-        <button class="rounded-button" @click="toggleBack">Back</button>
+        <button class="rounded-button" @click="toggleBack" aria-label="Go back to landing page">
```
Comment (HaydenNorris) [inline src/views/PlayView.vue:5]: some buttons have attributes on new lines, but this one has all on one. It's best to try to stick with 1 format - but this is just code styling.

File: src/views/PlayView.vue [hunk 3]
```diff
@@ -30,65 +44,73 @@ export default {
...
-    };
+    }
   },
   created() {
-    if (this.name === '') {
-      this.name = this.$userStore.name;
-    } 
-    this.$clueStore.clear();
-    this.$gameStore.clear();
+    this.$clueStore.clear()
```
Comment (HaydenNorris) [inline src/views/PlayView.vue:52]: nice tidy here

File: src/views/PlayView.vue [hunk 4]
```diff
@@ -30,65 +44,73 @@ export default {
...
+    this.$gameStore.clear()
+  },
+  computed: {
+    buttonClass() {
+      return (type) => {
+        return type === 'host'
```
Comment (HaydenNorris) [inline src/views/PlayView.vue:58]: while this is very clever - it is tricky to read! might be worth using the more boring if else statements for readability.

**Review by HaydenNorris — CHANGES_REQUESTED** [review]

File: src/views/PlayView.vue (hunk 4 above)
Comment (jordan-emma) [reply src/views/PlayView.vue:58]: this has been reverted back to if statements for readbility

File: src/views/PlayView.vue (hunk 2 above)
Comment (jordan-emma) [reply src/views/PlayView.vue:5]: formatting fixed

File: src/js/socket.js (hunk 1 above)
Comment (jordan-emma) [reply src/js/socket.js:3]: Unsure what you mean by this, my .env holds:
http://localhost:5000/

**Review by HaydenNorris — APPROVED** [review]

**Pull request merged**